docker/MXNet/Dockerfile

```
@@ -0,0 +1,93 @@
+FROM ubuntu:20.04
+USER root
+WORKDIR /
+
+# Setup timezone
+ENV TZ=Europe/Moscow
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+# Add user
+RUN useradd -ms /bin/bash openvino && \
+    chown openvino -R /home/openvino
+
+# Install dependencies
+ARG DEPENDENCIES="apt-utils \
+                  autoconf \
+                  sudo \
+                  vim \
+                  automake \
+                  build-essential \
+                  cmake \
+                  cpio \
+                  curl \
+                  dialog \
+                  gnupg2 \
+                  libdrm2 \
+                  libglib2.0-0 \
+                  lsb-release \
+                  libgtk-3-0 \
+                  libtool \
+                  python3-pandas \
+                  python3-pip \
+                  python3-setuptools \
+                  python3-dev \
+                  python3-venv \
+                  pciutils \
+                  libpython3.8 \
+                  udev \
+                  unzip \
+                  wget \
+                  git"
+RUN apt-get update && \
+    apt-get install -y -qq --no-install-recommends ${DEPENDENCIES} && \
+    rm -rf /var/lib/apt/lists/*
+
+# Install OpenVINO
+ENV INSTALL_DIR /opt/intel/openvino_2022
+ARG OV_SHORT_VERSION=2022.3
+ARG OV_LONG_VERSION="${OV_SHORT_VERSION}.0.9052.9752fafe8eb_x86_64"
+RUN sudo mkdir /opt/intel && \
+    wget https://storage.openvinotoolkit.org/repositories/openvino/packages/${OV_SHORT_VERSION}/linux/l_openvino_toolkit_ubuntu20_${OV_LONG_VERSION}.tgz \
+        -O /tmp/openvino.tgz --no-check-certificate --quiet && \
+    tar -zxvf /tmp/openvino.tgz -C /opt/intel/ && \
+    sudo ln -s /opt/intel/l_openvino_toolkit_ubuntu20_${OV_LONG_VERSION} $INSTALL_DIR && \
+    rm -rf /tmp/*
+
+# Install OpenVINO dependencies
+WORKDIR $INSTALL_DIR/install_dependencies
+RUN ls -la $INSTALL_DIR/install_dependencies && echo y | ./install_openvino_dependencies.sh
+RUN cat /root/.bashrc > tmp && echo 'source $INSTALL_DIR/setupvars.sh' > /root/.bashrc
+
+# Install openvino-dev
+ARG OV_SHORT_VERSION=2022.3
+WORKDIR /tmp/
+RUN pip3 install --upgrade pip
+RUN pip3 install openvino-dev==${OV_SHORT_VERSION}
+
+# Install pycocotools
+RUN pip3 install pycocotools
```

Можно объединить установку pip-пакетов
```suggestion
RUN pip3 install pycocotools docker PyYAML
```

---

Спасибо! Сделано.